src/plugins/controls/common/control_group/control_group_panel_diff_system.ts

```diff
@@ -26,6 +27,19 @@ export const genericControlPanelDiffSystem: DiffSystem = {
 export const ControlPanelDiffSystems: {
   [key: string]: DiffSystem;
 } = {
+  [RANGE_SLIDER_CONTROL]: {
+    getPanelIsEqual: (initialInput, newInput) => {
+      if (!deepEqual(omit(initialInput, 'explicitInput'), omit(newInput, 'explicitInput'))) {
+        return false;
+      }
+
+      const { value: valueA = ['', ''] }: Partial<RangeSliderEmbeddableInput> =
+        initialInput.explicitInput;
+      const { value: valueB = ['', ''] }: Partial<RangeSliderEmbeddableInput> =
+        newInput.explicitInput;
+      return isEqual(valueA, valueB);
+    },
+  },
```

Comment:
This fixes a bug where resetting the selections back to "undefined" / empty would cause unsaved changes:

**Before:**

https://github.com/elastic/kibana/assets/8698078/c7b3e622-c235-4701-8c2e-66a93f0e81ef


**After:**

https://github.com/elastic/kibana/assets/8698078/c3336031-be82-46d5-aaed-8830bc9f02df

Reply:
Nice to see these stability improvements!